scripts/provisioning/cleanupcmd.py

```diff
@@ -68,6 +68,15 @@ def process(self):
     """Main processing function."""
     sys.stdout.write(f"Processing {self.name} {self.url}\n")
     self.phase_prereqs_validate(self.name)
+
+    try:
+      sys.stdout.write("checking if ldap service is running or not...\n")
+      if not self.is_service_active("slapd"):
+      self.restart_service("slapd")
```

Comment:
stop slapd service at the end of cleanup and clear its logfile

Reply:
done